tools/clang/include/clang/SPIRV/ModuleBuilder.h

```diff
@@ -97,11 +97,15 @@ class ModuleBuilder {
                           uint32_t rhs);
 
   // \brief Creates an unconditional branch to the given target label.
-  void createBranch(uint32_t targetLabel);
+  // An OpLoopMerge instruction will also be created if both continueLabel
+  // and mergeLabel are not 0.
```

Comment:
What if mergeLabel is nonzero but continueLabel is zero?
If you know that doesn't happen, then say it's a constraint on the caller.